src/core/scene/a-scene.js

```diff
@@ -118,8 +124,15 @@ var AScene = module.exports = registerElement('a-scene', {
     addBehavior: {
       value: function (behavior) {
         var behaviors = this.behaviors;
-        if (behaviors.indexOf(behavior) !== -1) { return; }
-        behaviors.push(behavior);
+        for (var b in behaviors) {
```

Comment:
The body of the function is so short that I think in either case it will be optimised to the same native code. I'll switch to forEach for consistency